source/javascripts/controllers/awards_controller.coffee

```diff
@@ -46,3 +47,4 @@ EGA.controller "AwardsController", ($scope, $http, $sce, filterPipeline) ->
 
   # fetch awards
   $http.get("data/awards.json").then (result) -> $scope.awards = result.data
+
```

Comment:
I really don't care about new lines. It might even be at the very end of the list of things I care about.